frontend/src/modules/layout/components/menu/menu-header.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div
+    class="h-14 flex items-center transition group pl-5 pr-3"
+  >
+    <router-link to="/">
+      <img
+        key="logo"
+        class="h-6 !opacity-100"
+        src="/images/logo/crowd-mini.svg"
+        alt="crowd.dev logo"
+      />
+    </router-link>
```

Comment:
I'm not sure if the logic of the button and logo appearing is correct.
At least with the design below. Could you check with Nuno?
![Screenshot 2023-10-25 at 17 11 18](https://github.com/CrowdDotDev/crowd.dev/assets/20134207/6f353020-7502-41a0-9aa1-d52c235f7a4a)

Also, when hovered, the toggle button should have a darker color